x-pack/plugins/infra/public/components/logging/log_analysis_setup/setup_flyout/module_list_card.tsx

```diff
@@ -26,19 +27,46 @@ export const LogAnalysisModuleListCard: React.FC<{
   moduleStatus,
   onViewSetup,
 }) => {
+  const {
+    services: {
+      ml,
+      application: { navigateToUrl },
+      notifications: { toasts },
+    },
+  } = useKibanaContextForPlugin();
+
+  const [viewInMlLink, setViewInMlLink] = useState<string>('');
+
+  const getMlUrl = async () => {
```

Comment:
I think we can skip this check. If the ML module is not available I don't think the setup flyout will ever be mounted anyway.